src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final String DEFAULT_FIGURE_COLOR = Color.WHITE.name();
+    public static final double DEFAULT_FIGURE_RADIUS = 10.0;
+
+    public Figure getRandomFigure() {
+        int figureNumber = new Random().nextInt(FIGURE_COUNT);
+        switch (figureNumber) {
+            case 0:
+                return squareObject();
+            case 1:
+                return rightTriangleObject();
+            case 3:
+                return rectangleObject();
+            case 4:
+                return isoscelesTrapezoidObject();
+            default:
+                return circleObject();
+        }
+    }
+
+    public Figure getDefaultFigure() {
+        return new Circle(DEFAULT_FIGURE_COLOR, DEFAULT_FIGURE_RADIUS);
+    }
+
+    private Square squareObject() {
```

Comment:
An action is performed in a method, so the name of the method should describe what is being performed
```suggestion
    private Square prepareSquare() {
```
OR
```suggestion
    private Square getSquare() {
```